objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/ObjectRecognitionConfigHandler.java

```diff
@@ -323,11 +326,18 @@ boolean createImageRetriever(Map<String, ?> dataSourceConfig) {
     private boolean prepareCommunication(Map<String, ?> general) {
         int polling = -1; // initializing to an invalid input
         boolean queryable = false;
+        source.pool = new ThreadPoolExecutor(5, 5, 0l, TimeUnit.MILLISECONDS, new LinkedBlockingQueue<Runnable>(10));
```

Comment:
Not for this merge, but may make sense (_i.e._ enter an issue...) to allow a choice about what to happen when the queue gets full.  Should we just drop things (which is what we do now), or should we bias for more recent executions (make it a stack kinda thing, but not really).  This may not be that important because if you're running into queuing issues, you've underallocated/oversubscribed your connector.  But worth considering (we can always decide, "no.")